test/integration-new/cni/env_vars/env_vars_test.go

```diff
@@ -0,0 +1,134 @@
+package env_vars
+
+import (
+	"regexp"
+
+	"github.com/aws/amazon-vpc-cni-k8s/test/framework/resources/k8s/manifest"
+	k8sUtils "github.com/aws/amazon-vpc-cni-k8s/test/framework/resources/k8s/utils"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	AWS_VPC_ENI_MTU            = "AWS_VPC_ENI_MTU"
+	AWS_VPC_K8S_CNI_LOG_FILE   = "AWS_VPC_K8S_CNI_LOG_FILE"
+	AWS_VPC_K8S_CNI_VETHPREFIX = "AWS_VPC_K8S_CNI_VETHPREFIX"
+)
+
+var _ = Describe("cni env test", func() {
+
+	Context("CNI Environment Variables", func() {
+		It("Verifying that secondary ENI is created", func() {
+			nodes, err := f.K8sResourceManagers.NodeManager().GetAllNodes()
+			Expect(err).NotTo(HaveOccurred())
+
+			for _, node := range nodes.Items {
+				instanceId := k8sUtils.GetInstanceIDFromNode(node)
+				instance, err := f.CloudServices.EC2().DescribeInstance(instanceId)
+				Expect(err).NotTo(HaveOccurred())
+
+				len := len(instance.NetworkInterfaces)
+				Expect(len).To(BeNumerically(">=", 2))
+			}
+		})
+
+		It("Changing AWS_VPC_ENI_MTU and AWS_VPC_K8S_CNI_VETHPREFIX", func() {
+			currMTUVal := getEnvValueForKey(AWS_VPC_ENI_MTU)
+			Expect(currMTUVal).NotTo(Equal(""))
+
+			currVETHPrefix := getEnvValueForKey(AWS_VPC_K8S_CNI_VETHPREFIX)
+			Expect(currVETHPrefix).NotTo(Equal(""))
+
+			k8sUtils.AddEnvVarToDaemonSetAndWaitTillUpdated(f, DAEMONSET, NAMESPACE, DAEMONSET, map[string]string{
+				AWS_VPC_ENI_MTU:            "1300",
+				AWS_VPC_K8S_CNI_VETHPREFIX: "veth",
+			})
```

Comment:
I noticed @abhipth set the DS env var in the BeforeSuite, in pod_networking_suite_test.go: https://github.com/aws/amazon-vpc-cni-k8s/blob/master/test/integration-new/cni/pod_networking_suite_test.go#L80

Is there a reason why we're setting the ENV here?

Reply:
This test case is specifically about validating setting of these env vars. So setting it here. @abhipth needs that env config to be used in all his test cases. So he has done it in the BeforeSuite.